src/main/java/org/traccar/web/WebServer.java

```diff
@@ -100,6 +101,16 @@ protected void handleErrorPage(
         handlers.addHandler(servletHandler);
         handlers.addHandler(new GzipHandler());
         server.setHandler(handlers);
+
+        if (config.getBoolean("web.logRequests")) {
+          NCSARequestLog requestLog = new NCSARequestLog("./logs/jetty-yyyy_mm_dd.request.log");
+          requestLog.setAppend(true);
+          requestLog.setExtended(true);
+          requestLog.setLogTimeZone("GMT");
```

Comment:
Why GMT?

Reply:
standard

Reply:
Can you please provide a reference.

Reply:
I say this because I never saw a request log that it's not on utc. But we can't take it off and I suppose it will use the servers timezone.

Reply:
Let's remove it then, unless it's actually some kind of standard. I think a lot of servers are configured to be in UTC, so that might explain why you only see UTC.

Reply:
I removed it. No, that's not why, IIS for instance it's utc by default independently of the server timezone.